src/app/src/main/kotlin/org/vaccineimpact/api/app/controllers/ModellingGroupController.kt

```diff
@@ -87,11 +89,13 @@ open class ModellingGroupController(context: ControllerContext)
     {
         val touchstoneId = context.params(":touchstone-id")
         val groupId = context.params(":group-id")
-        val disease = context.getPart("disease").readText()
         context.checkEstimatePermissionsForTouchstone(groupId, touchstoneId, estimateRepository)
-        val description = context.getPart("description").readText()
 
-        val modelRuns = context.csvData<ModelRun>(RequestBodySource.HTMLMultipart("file"))
+        val parts = context.getParts()
+        val disease = parts.getPart("disease")
+        val description = parts.getPart("description")
+        val modelRuns = context.csvData<ModelRun>(parts.getPartAsRequestBodySource("file"))
+
```

Comment:
This seems like quite a convoluted way to read the data from the file. How about adding an overload to `context.csvData` that accepts a string instead of a request body source? Then could just call `val modelRuns = context.csvData<ModelRun>(parts.getPart("file"))` and it would be clearer what's going on here.

Reply:
Also I wonder if `parts.getPart` would be better named just `parts.get`, which feels semantically the same and distinguishes it more quickly from usages of `context.getPart`